### PR TITLE
HPCC-14223 Reword Regression Test Engine retry attempt message.

### PR DESCRIPTION
**Pull request by AttilaVamos**
Reword messages.

Remove hardwired timeout constants.

Signed-off-by: Attila Vamos attila.vamos@gmail.com

### CODE REVIEW

**Comment by AttilaVamos** [comment]
@richardkchapman please check and merge.

File: testing/regress/hpcc/regression/regress.py [hunk 1]
```diff
@@ -343,20 +343,21 @@ def StartTimeoutThread(self):
         self.timeoutThread.start()
 
     def CheckTimeout(self, cnt,  threadId,  query):
+        defSleepTime=0.1
```
Comment (richardkchapman) [inline testing/regress/hpcc/regression/regress.py]: Was this change (1.0 to 0.1) intentional ? desirable? It's not really part of the Jira ticket this PR is claiming to address.

Comment (AttilaVamos) [inline testing/regress/hpcc/regression/regress.py]: That change was intentional, but I can move it into a different JIRA/PR if necessary.

Comment (richardkchapman) [inline testing/regress/hpcc/regression/regress.py]: It's ok, just wanted to check.

**Pull request merged**